ci-operator/step-registry/ipi/install/nutanix/csi/ipi-install-nutanix-csi-commands.sh

```diff
@@ -133,36 +132,13 @@ allowVolumeExpansion: true
 reclaimPolicy: Delete
 EOF
 
-cat > "${SHARED_DIR}/manifest_iscsid-enable-master.yaml" << EOF
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
-metadata:
-  labels:
-    machineconfiguration.openshift.io/role: master
-  name: 99-master-ntnx-csi-enable-iscsid
-spec:
-  config:
-    ignition:
-      version: 3.2.0
-    systemd:
-      units:
-      - enabled: true
-        name: iscsid.service
-EOF
+oc apply -f "${SHARED_DIR}/manifest_0000-nutanix-csi-crd-manifest.yaml"
+oc apply -f "${SHARED_DIR}/manifest_0001-nutanix-csi-ntnx-system-namespace.yaml"
+oc apply -f "${SHARED_DIR}/manifest_0002-nutanix-csi-ntnx-secret.yaml"
+oc apply -f "${SHARED_DIR}/manifest_0003-nutanix-csi-operator-group.yaml"
+oc apply -f "${SHARED_DIR}/manifest_0004-nutanix-csi-subscription.yaml"
+oc apply -f "${SHARED_DIR}/manifest_0005-nutanix-csi-storage.yaml"
+oc apply -f "${SHARED_DIR}/manifest_0006-nutanix-csi-storage-class.yaml"
 
-cat > "${SHARED_DIR}/manifest_iscsid-enable-worker.yaml" << EOF
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
-metadata:
-  labels:
-    machineconfiguration.openshift.io/role: worker
-  name: 99-worker-ntnx-csi-enable-iscsid
-spec:
-  config:
-    ignition:
-      version: 3.2.0
-    systemd:
-      units:
-      - enabled: true
-        name: iscsid.service
-EOF
\ No newline at end of file
+sleep 60
+oc wait --for condition=Available=True --timeout=5m deployment/nutanix-csi-controller -n ntnx-system
```

Comment:
```suggestion
oc wait --for condition=Available=True --timeout=5m deployment/nutanix-csi-controller -n openshift-nutanix-csi
```
so that it would end up in `must-gather` archive

Reply:
@vrutkovs I've reverted changes to openshift-nutanix-csi. We need to change csi operator otherwise in openshift-* namespaces it produces 3 tests failures: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/36641/rehearse-36641-pull-ci-openshift-installer-master-e2e-nutanix-ovn/1630178035661541376
We'll make these changes after csi driver fix.

Reply:
This can be done later though